src/main/java/com/dumbdogdiner/stickyapi/common/util/commonmarkextensions/MCColorFormatBlockParser.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util.commonmarkextensions;
+
+import lombok.Getter;
+import org.bukkit.ChatColor;
+import org.commonmark.node.Block;
+import org.commonmark.parser.block.*;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MCColorFormatBlockParser extends AbstractBlockParser {

```

Comment:
What is this?

Reply:
Parser for markdown for rulebook, so that markdown can become json

Reply:
Could you perhaps be more explicit with the class name? In my humble opinion, `MCColorFormatBlockParser` isn't that close to `RulebookMarkdownParser`